testing/endtoend/endtoend_test.go

```diff
@@ -174,7 +175,7 @@ func (r *testRunner) waitForChainStart() {
 }
 
 // runEvaluators executes assigned evaluators.
-func (r *testRunner) runEvaluators(conns []*grpc.ClientConn, tickingStartTime time.Time) error {
+func (r *testRunner) runEvaluators(conns []validatorHelpers.NodeConnection, tickingStartTime time.Time) error {
```

Comment:
revert this change

Reply:
@PatriceVignola That is just using the grpc client to access the related rpc methods from the beacon node. They do not access the validator Ex:
```
StreamBlocks , StreamBlocksAltair
```

are methods that are served by the beacon node. Whether you are running with the rest-api or not makes no difference here.

Reply:
Ah I see what you mean now. I was confused by the calls to `GetDuties()` and `DomainData()` in the `slashing.go` file, which both are endpoints that we intend to convert to REST. Sorry for the confusion! I went ahead and reverted the changes to the evaluators, it should look much better now.